packages/realm-react/example/.prettierrc.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  singleQuote: true,
+  trailingComma: 'all',
+  arrowParens: 'avoid',
```

Comment:
I just left these standard to the react native template output.  We could remove all the eslint and prettier configs here, but then would have to fix the warnings afterwards, since we have stricter standards in the root project.

Reply:
fixed